swlor2_tlk/swlor2_tlk.tlk.json

```diff
@@ -64872,6 +64872,27 @@
     {
       "id": 192500,
       "text": "[CEP] Door 1500"
+    },
+    {
```

Comment:
Where are the entries for Rodians? Maybe this is in the swlor2_tlk.tlk file. Note: the JSON form is primarily for our reference during pull requests so it might not be in sync if you didn't run the ConvertTlk.cmd script in SWLOR_Haks. Please run this and update the PR so it gets back in sync - this is something new I neglected to mention.